src/it/basic/pom.xml

```diff
@@ -0,0 +1,65 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.massfords.test</groupId>
+    <artifactId>jaxb-basic</artifactId>
+    <version>1.0</version>
+    <properties>
+        <maven.test.skip>true</maven.test.skip>
+    </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>com.evolvedbinary.maven.jvnet</groupId>
+                <artifactId>jaxb30-maven-plugin</artifactId>
+                <version>0.15.0</version>
+                <configuration>
+                    <strict>false</strict>
+                    <extension>true</extension>
+                    <schemaDirectory>
+                        ${basedir}/src/main/resources
+                    </schemaDirectory>
+                    <args>
+                        <arg>-Xvisitor</arg>
+                        <arg>-Xvisitor-package:org.example.visitor</arg>
+                    </args>
```

Comment:
lots of duplication in the test files since each test has its own pom

the pom files are mostly the same apart from the schemaDirectory and args to the plugin

I'll look at reducing the amount of config for the tests in another pass